### PR TITLE
Add host inventory metrics to azure compute_vm metricset

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to add proposed host common fields into `compute` metricset:
- host.id
- host.name
- host.cpu.pct
- host.network.in.bytes
- host.network.in.packets
- host.network.out.bytes
- host.network.out.packets
- host.disk.read.bytes
- host.disk.write.bytes

## Why is it important?

This PR is a part of the inventory schema work. 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## How to test this PR locally
1. Start Metricbeat `azure` module: `./metricbeat modules enable azure`
2. Edit `modules.d/azure.yml` to only include `compute_vm` metricset:
```
- module: azure
  metricsets:
    - compute_vm
  enabled: true
  period: 300s
  client_id: '123'
  client_secret: '123'
  tenant_id: '123'
  subscription_id: '123'
  refresh_list_interval: 600s
```
3. Change `add_host_metadata` processor config in metricbeat.yml file:
```
processors:
  - add_host_metadata:
      replace_fields: false
```
4. Start metricbeat
5. You should see metrics from `compute_vm` metricset and includes fields listed above.

### CODE REVIEW

File: x-pack/metricbeat/module/azure/compute_vm/compute_vm_integration_test.go [hunk 1]
```diff
@@ -2,9 +2,6 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
```
Comment (narph) [inline x-pack/metricbeat/module/azure/compute_vm/compute_vm_integration_test.go:4]: integration headers are missing

Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/azure/compute_vm/compute_vm_integration_test.go:4]: ahh sorry I removed them for testing and forgot to add them back 😄 Will do!

Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/azure/compute_vm/compute_vm_integration_test.go]: done.

**Review by narph — APPROVED** [review]
LGTM

**Pull request merged**